src/components/Footer.js

```diff
@@ -1,7 +1,7 @@
 export default function Footer() {
   return (
     <>
-      <footer className='w-full h-24 flex justify-center items-center'>
+      <footer className='w-screen h-24 flex justify-center items-center'>
```

Comment:
Let's get rid of the default text of the footer and just make it "made with netlify" or an unfoldingWord footer or no footer.